components/AppBar.tsx

```diff
@@ -0,0 +1,48 @@
+'use client';
+
+import {
+  AppBar as MuiAppBar,
+  Stack,
+  SxProps,
+  Theme,
+  Toolbar,
+  Typography,
+} from '@mui/material';
+import Image from 'next/image';
+import Link from 'next/link';
+
+const APP_BAR_STYLES: SxProps<Theme> = {
+  backgroundColor: 'common.white',
+  color: 'primary.dark',
```

Comment:
@terotik @lilia1891 When using the `sx` styling approach, properties that support "theme aware" strings are documented here: https://mui.com/system/getting-started/the-sx-prop/#theme-aware-properties.

In this example, `backgroundColor` is a theme aware property which represents the colour in `theme.palette.common.white`